backend/src/main/java/reviewme/review/service/ReviewListLookupService.java

```diff
@@ -0,0 +1,42 @@
+package reviewme.review.service;
+
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+import reviewme.review.domain.exception.ReviewGroupNotFoundByReviewRequestCodeException;
+import reviewme.review.service.dto.response.list.ReceivedReviewsResponse;
+import reviewme.review.service.dto.response.list.ReviewListElementResponse;
+import reviewme.review.service.exception.ReviewGroupUnauthorizedException;
+import reviewme.review.service.module.ReviewListMapper;
+import reviewme.reviewgroup.domain.ReviewGroup;
+import reviewme.reviewgroup.repository.ReviewGroupRepository;
+
+@Service
+@RequiredArgsConstructor
+public class ReviewListLookupService {
+
+    private final ReviewGroupRepository reviewGroupRepository;
+    private final ReviewListMapper reviewListMapper;
+
+    @Transactional(readOnly = true)
+    public ReceivedReviewsResponse getReceivedReviews(String reviewRequestCode, String groupAccessCode) {
+        ReviewGroup reviewGroup = findReviewGroupByRequestCodeOrThrow(reviewRequestCode);
+        validateGroupAccessCode(groupAccessCode, reviewGroup);
+
+        List<ReviewListElementResponse> reviewGroupResponse = reviewListMapper.mapToReviewList(reviewGroup);
+        return new ReceivedReviewsResponse(reviewGroup.getReviewee(), reviewGroup.getProjectName(),
+                reviewGroupResponse);
+    }
+
+    private ReviewGroup findReviewGroupByRequestCodeOrThrow(String reviewRequestCode) {
+        return reviewGroupRepository.findByReviewRequestCode(reviewRequestCode)
+                .orElseThrow(() -> new ReviewGroupNotFoundByReviewRequestCodeException(reviewRequestCode));
+    }
+
+    private static void validateGroupAccessCode(String groupAccessCode, ReviewGroup reviewGroup) {
```

Comment:
```suggestion
    private void validateGroupAccessCode(ReviewGroup reviewGroup, String groupAccessCode) {
```
- `static`이 붙어 있네요 👀 
- 그룹의 코드를 검증하니 순서를 이렇게 바꾸면 가독성이 더 좋아진다고 느껴져요.
- (개인적인 의견) 한 번밖에 쓰이지 않고, `if`문 안이 명확해서 메서드로 뽑지 않아도 괜찮지 않을까요? 이건 다른 크루들 의견도 들어보고 싶어요

Reply:
@donghoony 저는 둘 다 괜찮다 주의입니다..!

Reply:
개인적으로 검증 책임이 있는 메서드는 한번만 쓰이더라도 분리하는 것이 보기 편한 것 같습니다! 다른 부분은 반영 했습니다 👍